src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java

```diff
@@ -67,12 +76,23 @@ protected OutputSettingsAudio() {
         add(metadataOption);
     }
 
+    /**
+     * Sets all possible quality for Audio converter.
+     */
+    protected void setQualitySelect() {
+        qualitySelect.addItem("");
+        qualitySelect.addItem("64");
+        qualitySelect.addItem("128");
+        qualitySelect.addItem("192");
+        qualitySelect.addItem("320");
```

Comment:
Magic numbers, would be better to set them to final attributes, so they are more understandable.